js/ParticipantTopDonor.js

```diff
@@ -10,10 +10,11 @@
       // First, sum all the donations for each name.
       var groupedData = {};
       $.each(data, function(i, val) {
+        var name = val.donorName || "Anonymous";
         if (groupedData.name !== undefined) {
```

Comment:
is this if necessary? aren't we doing the same thing in both branches? or am i crazy @_@